src/PJV.test.ts

```diff
@@ -1,9 +1,11 @@
 import { assert, describe, it, test } from "vitest";
 
-import { PJV } from "./PJV.js";
+import { PJV } from "./PJV";
 
-function getPackageJson(extra) {
-	const out = {
+const getPackageJson = (
+	extra: Record<string, any> = {},
+): Record<string, any> => {
+	const out: Record<string, any> = {
```

Comment:
[Style] This should be linted to:

```suggestion
	extra: Record<string, unknown> = {},
): Record<string, unknown> => {
	const out: Record<string, unknown> = {
```